src/GraphQL.NewtonsoftJson/StringExtensions.cs

```diff
@@ -50,6 +53,38 @@ public static Inputs ToInputs(this JObject obj)
             return GetValueInternal(values) as Dictionary<string, object>;
         }
 
+        /// <summary>
+        /// Deserializes a JSON-formatted string of data into the specified type.
+        /// Any <see cref="Inputs"/> objects will be deserialized into the proper format.
+        /// </summary>
+        public static T FromJson<T>(this string json)
+        {
+            var settings = new JsonSerializerSettings
```

Comment:
This can be moved into static JsonSerializer field:
```
var settings = new JsonSerializerSettings
            {
                DateFormatHandling = DateFormatHandling.IsoDateFormat,
                DateParseHandling = DateParseHandling.None
            }
 settings.Converters.Add(_inputsConverter);
 JsonSerializer jsonSerializer = JsonSerializer.Create(settings); <-- jsonSerializer  may be static
```
`JsonConvert.DeserializeObject` is just a wrapper that allocates.

Reply:
The same for `ToDictionary(this string json)`

Reply:
Done